.github/workflows/release-docs.yml

```diff
@@ -65,6 +67,13 @@ jobs:
             echo "Creating new docs branch ${DOCS_BRANCH}"
           fi
           echo "PR_TITLE=${PR_TITLE:-}" >> $GITHUB_OUTPUT
+
+      - name: Update the technical docs link in the nav.adoc file
+        run: |
+          MAJOR_MINOR="${{ steps.validate_tag.outputs.MAJOR }}-${{ steps.validate_tag.outputs.MINOR }}"
+          echo "Major and minor version extracted: $MAJOR_MINOR"
+          sed -i '' -E "s|(https://)[^/]*openzeppelin-relayer\.netlify\.app|\1docs-v${MAJOR_MINOR}--openzeppelin-relayer.netlify.app|g" docs/modules/ROOT/nav.adoc
```

Comment:
Lets make sure to change it to `release-v*` version and then on push to docs branch change it to `docs-v*`